sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ApplyDefaultCollation.scala

```diff
@@ -103,13 +124,12 @@ object ApplyDefaultCollationToStringType extends Rule[LogicalPlan] {
    * Determines the default collation for an object in the following order:
    * 1. Use the object's explicitly defined default collation, if available.
    * 2. Otherwise, use the default collation defined by the object's schema.
```

Comment:
```suggestion
   * 2. Otherwise, use the default collation defined by the object's schema.
   * 3. If not defined in the schema, use the default collation from the object's catalog.
```